pythonforandroid/recipes/libxslt/__init__.py

```diff
@@ -0,0 +1,23 @@
+from pythonforandroid.toolchain import (CythonRecipe, shprint,
+                                        current_directory, info, Recipe)
+from pythonforandroid.patching import will_build, check_any
+import sh
+from os.path import exists,join
+
+class LibxsltRecipe(CythonRecipe):
+    version = '1.1.29'
+    url = 'ftp://xmlsoft.org/libxslt/libxslt-{version}.tar.gz'
+    name = 'libxslt'
+    depends = [('python2', 'python3crystax'), 'libxml2']
+    patches = ['fix-dlopen.patch']
+
+    def should_build(self, arch):
+        return not exists(join(self.get_build_container_dir(arch.arch),"libxslt/.libs/libxslt.a"))
```

Comment:
I think you can use `self.get_build_dir(arch.arch)` and remove the `libxslt` folder from the path - this should work the same, but will also work if the extracted libxslt folder is something unexpected. This is a small thing though, it probably isn't important to having the recipe actually work.